pycarol/carol.py

```diff
@@ -1,515 +1,470 @@
-                if __count < 5: #To avoid infinity loops
+                if __count < 5:  # To avoid infinity loops
                     continue
-                else:
-                    raise Exception('Too many retries to refresh token.\n', response.text, response.status_code)
+
+                raise Exception(
+                    "Too many retries to refresh token.\n",
+                    response.text,
+                    response.status_code,
+                )
             elif response.status_code == 404:
-                raise CarolApiResponseException(response.text, response.status_code)
+                raise exceptions.CarolApiResponseException(
+                    response.text, response.status_code
+                )
 
             raise Exception(response.text, response.status_code)
 
-    def issue_api_key(self, connector_id=None):
-        """
-        Create an API key for a given connector.
+    def issue_api_key(self, connector_id: T.Optional[str] = None) -> ResponseType:
+        """Create an API key for a given connector.
 
         Args:
+            connector_id: Connector ID to be used when creating the APIkey
 
-            connector_id: `str` default `None`.
-                Connector ID to be used when creating the APIkey
-
-        Returns: `dict`
+        Returns:
             Dictionary with the API key.
-
         """
-
         if connector_id is None:
             connector_id = self.connector_id
-        resp = self.call_api('v2/apiKey/issue', data={
-            'connectorId': connector_id
-        }, content_type='application/x-www-form-urlencoded')
+        resp = self.call_api(
+            "v2/apiKey/issue",
+            data={"connectorId": connector_id},
+            content_type="application/x-www-form-urlencoded",
+        )
         return resp
 
-    def api_key_details(self, api_key, connector_id):
-
-        """
-        Display information about the API key.
+    def api_key_details(self, api_key: str, connector_id: str) -> ResponseType:
+        """Display information about the API key.
 
         Args:
+            api_key: Carol's api key
+            connector_id: Connector Id which API key was created.
 
-            api_key: `str`
-                Carol's api key
-            connector_id: `str`
-                Connector Id which API key was created.
-
-        Returns: `dict`
+        Returns:
             Dictionary with API key information.
-
         """
-
-        resp = self.call_api('v2/apiKey/details',
-                             params = {"apiKey": api_key, "connectorId": connector_id})
+        resp = self.call_api(
+            "v2/apiKey/details", params={"apiKey": api_key, "connectorId": connector_id}
+        )
 
         return resp
 
-    def api_key_revoke(self, connector_id):
-        """
-        Revoke API key for ta given connector_id
+    def api_key_revoke(self, connector_id: str) -> ResponseType:
+        """Revoke API key for ta given connector_id.
```

Comment:
"Revoke API key for ta? given connector_id."

"ta" é um typo pra "a" ou uma abreviação pra "tenant"?

Reply:
corrigido: https://github.com/totvslabs/pyCarol/commit/8161f712af112ff74c04137664d9ac4dc5bc7d05